apps/api/src/routers/user.py

```diff
@@ -89,13 +89,17 @@ async def apply(
         raise HTTPException(status.HTTP_403_FORBIDDEN, "Applications have closed.")
 
     # check if email is already in database
-    EXISTING_RECORD = await mongodb_handler.retrieve_one(
-        Collection.USERS, {"_id": user.uid}
+    existing_record = await mongodb_handler.retrieve_one(
+        Collection.USERS, {"_id": user.uid, "roles": {"exists": True}}, ["roles"]
```

Comment:
Note: I made a mistake here, the query should use `"$exists"`, not just `"exists"`